src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -59,16 +57,16 @@ void CheckActivationCorrect(const arma::colvec input,
   // Test the activation function using a single value as input.
   for (size_t i = 0; i < target.n_elem; ++i)
   {
-    BOOST_REQUIRE_CLOSE(ActivationFunction::Fn(input.at(i)),
-        target.at(i), 1e-3);
+    REQUIRE(ActivationFunction::Fn(input.at(i)) ==
+        Approx(target.at(i)).epsilon(1e-3));
```

Comment:
```suggestion
    REQUIRE(ActivationFunction::Fn(input.at(i)) ==
        Approx(target.at(i)).epsilon(1e-3 / 100));
```

Reply:
Ahhh, I misunderstood the catch tolerance.  Thanks for catching this (no pun intended :smile:).  I've updated all the tolerances to divide by 100 in dfdd9e764.